core/templates/pages/contributor-dashboard-page/contributions-and-review/contributions-and-review.component.ts

```diff
@@ -129,7 +131,9 @@ export class ContributionsAndReview
     };
   };
 
-  accomplismentsEnabled: boolean = false;
+  TAB_TYPES_TO_NAMES = {};
+
+  accomplishmentsTabIsEnabled: boolean = false;
```

Comment:
Add a docstring here to explain what this is for ... tie it to the feature flag and give the name of that flag.